### ISSUE TITLE
check for version of perl-XML-Simple in OSCAR 4.2.1

### ISSUE THREAD

**Issue by dikim33** [oscar-cluster/oscar#166]
**Reported by  on 20 Jan 2006 00:59 UTC**
```
it seems that currently OSCAR 4.2.1 to install
perl-XML-Simple just checks to see if it is installed,
and does not actually check the version. Thus if
installing OSCAR 4.2 on top of a previous version of
OSCAR, the following error appears:
> error: Failed dependencies:
>        perl(XML::Simple) >= 2.08 is needed by
> systemimager-server-3.5.3-3oscar.noarch
> $pm->install ($dm->query_required_by ()) failed at
./wizard_prep line 312
> Couldn't install packages needed for OSCAR Wizard to
run at
> ./wizard_prep line 312
> Oscar Wizard preparation script failed to complete at
./install_cluster
> line 212.
```

**Comment by dikim33** [comment]
**Comment by efocht on 4 Aug 2006 09:06 UTC**
Bernard asked:
Erich - is this still an issue with the current system with yum(e)?  Do we need to hardcode the require of perl-XML-Simple >= 2.08 in perl-XML-Simple's prereq.cfg?

As install_prereq isn't able to deal with version comparisons, yet, this won't work. But I think during the smart install of systemimager the perl-XML-Simple rpm should get updated. I'll try to check.

**Comment by dikim33** [comment]
**Comment by efocht on 13 Sep 2006 10:07 UTC**
This will be completely solved when the feature request #208 is implemented.

Currently perl-XML-Simple is installed "smartly", therefore even if an older
version is installed, it should be updated during intsallation with the newer version from the OSCAR repository.
